packages/dx-react-chart-demos/src/demo-data/data-vizualization.js

```diff
@@ -25,6 +25,212 @@ export const born = [
   },
 ];
 
+export const australianMedals = [
+  {
+    year: 1896, gold: 2, silver: null, bronze: null,
+  }, {
+    year: 1900, gold: 2, silver: null, bronze: 3,
+  }, {
+    year: 1904, gold: null, silver: null, bronze: null,
+  }, {
+    year: 1908, gold: 1, silver: 2, bronze: 2,
+  }, {
+    year: 1912, gold: 2, silver: 2, bronze: 3,
+  }, {
+    year: 1916, gold: null, silver: null, bronze: null,
+  }, {
+    year: 1920, gold: null, silver: 2, bronze: 1,
+  }, {
+    year: 1924, gold: 3, silver: 1, bronze: 2,
+  }, {
+    year: 1928, gold: 1, silver: 2, bronze: 1,
+  }, {
+    year: 1932, gold: 3, silver: 1, bronze: 1,
+  }, {
+    year: 1936, gold: null, silver: null, bronze: 1,
+  }, {
+    year: 1940, gold: null, silver: null, bronze: null,
+  }, {
+    year: 1944, gold: null, silver: null, bronze: null,
+  }, {
+    year: 1948, gold: 2, silver: 6, bronze: 5,
+  }, {
+    year: 1952, gold: 6, silver: 2, bronze: 3,
+  }, {
+    year: 1956, gold: 13, silver: 8, bronze: 14,
+  }, {
+    year: 1960, gold: 8, silver: 8, bronze: 6,
+  }, {
+    year: 1964, gold: 6, silver: 2, bronze: 10,
+  }, {
+    year: 1968, gold: 5, silver: 7, bronze: 5,
+  }, {
+    year: 1972, gold: 8, silver: 7, bronze: 2,
+  }, {
+    year: 1976, gold: 0, silver: 1, bronze: 4,
```

Comment:
Somewhere you are using `null` somewhere `0`. What is a difference?

Reply:
The difference only in the calculation of range and rendering of axis resp: from zero or from another non-zero value. I'll change all values to zero.